devlab/tests/filtering_utils.py

```diff
@@ -69,6 +77,34 @@ def filter_tenants(self, src_data_list):
                 src_data_list.pop(index)
         return [src_data_list, popped_tenant_list]
 
+
+class MigrationUtils(object):
+
+    def __init__(self):
+        self.username_for_ssh = 'cirros'
+        self.password_for_ssh = 'cubswin:)'
+
+    def execute_command_on_vm(self, ip_addr, cmd, username=None,
+                              warn_only=False, password=None, key=None,
+                              use_sudo=True):
+
+        if username is None:
+            username = self.username_for_ssh
+        if password is None and key is None:
+            password = self.password_for_ssh
+        with hide('output', 'running', 'warnings'), settings(
```

Comment:
I assume, you can replace the string
hide('output', 'running', 'warnings')
with
hide('everything')